fabric/v4/model_service_profile_filter.go

```diff
@@ -9,4 +9,4 @@
  */
 package v4
 
-type ServiceProfileFilter struct{}
+type ServiceProfileFilter interface{}
```

Comment:
This is a generated file. Any changes should be made through API spec changes or (in this case) post generated code patches (like https://github.com/equinix-labs/fabric-go/blob/main/patches/post/02_fix-updatevalue.patch).

The current body of https://patch-diff.githubusercontent.com/raw/equinix-labs/fabric-go/pull/3.patch could serve as your patch.

Reply:
I want to make sure I'm clear with the suggestion offered.

Create a file in `patches/post/` called `03_service_profile_filter_interface.patch`. The contents should be:

```patch
From 1f92ba768c8f3ac5597057a59ab294ecbb36aa39 Mon Sep 17 00:00:00 2001
From: garisingh <garisingh@equinix.com>
Date: Mon, 8 Aug 2022 15:29:31 -0700
Subject: [PATCH] Modified Service Profile empty struct to Interface

---
 fabric/v4/model_service_profile_filter.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fabric/v4/model_service_profile_filter.go b/fabric/v4/model_service_profile_filter.go
index b95682e..afa2d8b 100644
--- a/fabric/v4/model_service_profile_filter.go
+++ b/fabric/v4/model_service_profile_filter.go
@@ -9,4 +9,4 @@
  */
 package v4
 
-type ServiceProfileFilter struct{}
+type ServiceProfileFilter interface{}
```